pyfarm/master/api/tasklogs.py

```diff
@@ -163,7 +163,7 @@ def post(self, job_id, task_id, attempt):
                            error="Specified task not found"), NOT_FOUND
 
         path = join(LOGFILES_DIR, g.json["identifier"])
-        if not realpath(path).startswith(LOGFILES_DIR):
+        if not abspath(path).startswith(LOGFILES_DIR):
```

Comment:
On a mac, this fixes a bug where `path = join(LOGFILES_DIR, g.json["identifier"])` would not be the start of `realpath(path)` because `/tmp` is a symlink.

You're right however, this should and will be in another change.  And, the better solution is to apply `abspath` to `path` instead I think after looking at this again...